### PR TITLE
[Python PQ/RQ] Utilities

### PR DESCRIPTION
**Pull request by sebaraba**
## Done:

- [x] doctests
- [x] schedule-evenets-with-sched

### CODE REVIEW

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - python/core/utilities-i/schedule-events-with-sched.md

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - python/core/utilities-i/schedule-events-with-sched.md

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - python/core/utilities-i/schedule-events-with-sched.md

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/python/core/testing/doctests.md?ref=d93d49ce2650adb21c6516178ef4ddfe3d23a89f
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/python/core/utilities-i/schedule-events-with-sched.md?ref=d93d49ce2650adb21c6516178ef4ddfe3d23a89f

File: python/core/utilities-i/schedule-events-with-sched.md [hunk 1]
```diff
@@ -56,3 +56,45 @@ Then we run our scheduler:
...
+## Practice
+
+Create a new schedule and set it to enter second with priority = 1 and delay = 4.
+
+```python
+
+import sched
+import time
+
+start = 2018-03-23 21:55:19.629694
```
Comment (lizTheDeveloper) [inline python/core/utilities-i/schedule-events-with-sched.md]: wait is this... how this actually works? without quotes and everything? I have never seen dates written like this, can you find an example (or fix it)

**Pull request merged**